src/models/blockchain/transaction/mod.rs

```diff
@@ -80,6 +80,15 @@ impl TransactionProof {
         Self::SingleProof(Proof(vec![]))
     }
 
+    /// A proof that will always be invalid, with a specified size measured in
+    /// number of [`BFieldElement`](twenty_first::math::b_field_element::BFieldElement)s.
+    #[cfg(test)]
+    pub(crate) fn invalid_single_proof_of_size(size: usize) -> Self {
+        use tasm_lib::twenty_first::bfe;
+
+        Self::SingleProof(Proof(vec![bfe!(0); size]))
+    }
+
```

Comment:
Please do not intersperse test with non-test functions. Move this function into the test module instead.